testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -0,0 +1,56 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
You are a hero.

We should have special "Software Sustainability Hero" badges for people contributing new tests!